cfep-02.md

```diff
@@ -0,0 +1,85 @@
+
+<table>
+<tr><td> Title </td><td> Upgrade default macOS Travis image to `xcode6.4` </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Andreas Kloeckner &lt;inform@tiker.net&gt;</td></tr>
+<tr><td> Created </td><td> Nov 8, 2016</td></tr>
+<tr><td> Updated </td><td> Nov 8, 2016</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/6 </td></tr>
+<tr><td> Implementation </td><td> (listed below) </td></tr>
+</table>
+
+## Abstract
+
+Upgrade the default Travis image to `xcode6.4`.
+
+## Implementation
+
+*   Adapt `conda smithy` so that it applies that edit on rerender.
+    [One-liner change here](https://github.com/conda-forge/conda-smithy/blob/b1d1730c4b8c8dc849464b1b39569b0e61ec3130/conda_smithy/templates/travis.yml.tmpl#L12)
+*   Apply the change in the `staged-recipes` master branch.
+    [One-liner change here](https://github.com/conda-forge/staged-recipes/blob/3ce42e083cebe5f39eea82d069d5a94cb7719218/.travis.yml#L5)
+
+    [PR](https://github.com/conda-forge/staged-recipes/pull/1094)
+*   Rebuild all packages with this? (TBD? Opinions?)
+
+    Likely not necessary given [C++ ABI Compatibility][cxxabi].
+*   Suggest that all new packages include the `toolchain` so that
```

Comment:
I am against the use of any un-versioned package that can change the build characteristics like that. We already have have feedstocks that have to override those variables to build properly. IMO those belong in the recipe and are, ultimately, the maintainers decision.

With that said I no longer have a stake on this as we diverged from conda-forge a long time ago, since the toolchain was first added. Unfortunately most of earth-science software we build locally must use `gcc` and must provide `libgomp` and others at run time to ensure they will run anywhere.